articles/service-fabric/scripts/service-fabric-powershell-change-rdp-user-and-pw.md

```diff
@@ -39,6 +39,9 @@ This script uses the following commands: Each command in the table links to comm
 | [Add-AzVmssExtension](/powershell/module/az.compute/add-azvmssextension)| Adds an extension to the virtual machine scale set.|
 | [Update-AzVmss](/powershell/module/az.compute/update-azvmss)|Updates the state of a virtual machine scale set to the state of a local VMSS object.|
 
+##Duration
+A single NodeType with 5 nodes for example has a duration of 45 a 60 minutes to change username and / or password. 
```

Comment:
This is going to be dependent on the durability level of the VMSS. For example it is possible for a node type which is running a VMSS on silver durability to take up to 15 minutes per upgrade domain. This update would then require 5 * 15 = 75 minutes to get through all of the upgrade domains.